tests/bluemira/equilibria/fem_fixed_boundary/test_fem_magnetostatic_2D.py

```diff
@@ -0,0 +1,139 @@
+# bluemira is an integrated inter-disciplinary design tool for future fusion
+# reactors. It incorporates several modules, some of which rely on other
+# codes, to carry out a range of typical conceptual fusion reactor design
+# activities.
+#
+# Copyright (C) 2021 M. Coleman, J. Cook, F. Franza, I.A. Maione, S. McIntosh, J. Morris,
+#                    D. Short
+#
+# bluemira is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# bluemira is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with bluemira; if not, see <https://www.gnu.org/licenses/>.
+
+import os
+
+import dolfin
+import matplotlib.pyplot as plt
+import numpy as np
+
+import bluemira.geometry.tools as tools
+from bluemira.base.components import Component, PhysicalComponent
+from bluemira.base.file import get_bluemira_root
+from bluemira.equilibria.fem_fixed_boundary.fem_magnetostatic_2D import (
+    FemMagnetostatic2d,
+)
+from bluemira.equilibria.fem_fixed_boundary.utilities import ScalarSubFunc, b_coil_axis
+from bluemira.geometry.face import BluemiraFace
+from bluemira.geometry.placement import BluemiraPlacement
+from bluemira.mesh import meshing
+
+HAS_MSH2XDMF = False
+try:
+    from bluemira.utilities.tools import get_module
+
+    msh2xdmf = get_module(
+        os.path.join(get_bluemira_root(), "..", "msh2xdmf", "msh2xdmf.py")
+    )
+
+    HAS_MSH2XDMF = True
+except ImportError as err:
+    print(f"Unable to import msh2xdmf, dolfin examples will not run: {err}")
```

Comment:
Actually, is `HAS_MSH2XDMF` used in this file? 

Also, I would use an `ImportWarning` here, as `pytest` will swallow the print statement.

Reply:
import removed

Reply:
I was wondering if this try-except is really necessary. If the module is not imported, the test should just fail. So I was thinking to remove the try-except and just leaving the importing code for msh2xdmf. What do you think about it?

Reply:
further clean-up expected after #855